docs/openstack/create-and-connect-to-the-VM/ssh-to-the-VM.md

```diff
@@ -0,0 +1,320 @@
+    and user sessions, you can select any option from the list of available display
+    managers. For instance, here we have `gdm3` as the default selection.
+
+    ![xrdp Display Manager](images/xrdp_display_manager.png)
+
+### Installing xrdp
+
+To install xrdp, run the following command in the terminal:
+
+    sudo apt install xrdp -y
+
+After installing xrdp, verify the status of xrdp using systemctl:
+
+    sudo systemctl status xrdp
+
+This command will show the status as **active (running)**:
+
+Output:
+
+    ● xrdp.service - xrdp daemon
+        Loaded: loaded (/lib/systemd/system/xrdp.service; enabled; vendor preset: enab>
+        Active: active (running) since Mon 2024-02-12 21:33:01 UTC; 9s ago
+        ...
+        CGroup: /system.slice/xrdp.service
+                └─8839 /usr/sbin/xrdp
+
+!!! info "What if xrdp is not Running?"
+    If the status of xrdp is not running, you may have to start the service manually
+    with this command: `sudo systemctl start xrdp`. After executing the above command,
+    verify the status again to ensure xrdp is in a running state.
+
+Make xrdp to use the desktop environment we previously created:
+
+    sudo sed -i.bak '/fi/a #xrdp multiple users configuration \n xfce-session \n' /etc/xrdp/startwm.sh
+
+### Configuring xrdp and Updating Security Groups
+
+If you want to customize the default xrdp configuration (optional), you will need
+to review the default configuration of xrdp, which is stored under `/etc/xrdp/xrdp.ini`.
+`xrdp.ini` is the default configuration file to set up RDP connections to the
+xrdp server. The configuration file can be modified and customized to meet the
+RDP connection requirements.
+
+Add a new security group with a RDP (port 3389) rule open to the public for an
+RDP connection and attach that security group to your instance.
+
+!!! info "How to Update Security Group(s) on a Running VM?"
+    Following [this guide](../access-and-security/security-groups.md#update-security-groups-to-a-running-vm),
+    you'll be able to attach any newly created security group(s) with all the
+    required rules to a running VM.
+
+Restart once the xrdp server to make sure all the above changes are reflected:
+
+    sudo systemctl restart xrdp
+
+### Testing the RDP Connection
+
+Now, you should now be able to connect to the Ubuntu VM via xrdp.
+
+#### Testing the RDP Connection on Windows
+
+If you are using Windows as a local desktop, Windows users have an RDP connection
+application by default on their machines.
+
+Enter your VM's Floating IP and username into the fillable text boxes for Computer
+and User name. You may need to press the down arrow for Show Options to input the
+username i.e. `ubuntu`:
+
+![RDP Windows](images/rdp_windows_for_xrdp.png)
+
+Press the Connect button. If you receive an alert that the Remote Desktop can't
+connect to the remote computer, check that you have properly attached the security
+group with a RDP (port 3389) rule open to the public to your VM.
```

Comment:
it feels like it might help to link to how to do this.

Reply:
Added a new section on Security Group page do describe about adding RDP

Reply:
that will be linked from here